packages/editor/src/components/post-schedule/style.scss

```diff
@@ -16,6 +16,11 @@
 	height: auto;
 	min-height: $button-size-compact;
 
+	// This needs specificity.
+	&.components-button {
+		text-wrap: pretty;
```

Comment:
I wonder if we should apply this to all buttons in `.editor-post-panel__row`? That way we'd also capture templates with long labels.

Reply:
We might fix this too:

<img width="290" alt="Screenshot 2024-05-08 at 15 18 09" src="https://github.com/WordPress/gutenberg/assets/846565/52190634-ebac-4193-9334-946b3a31223b">